.golangci.yml

```diff
@@ -17,6 +17,7 @@ linters:
     - wsl      # WSL is disabled, since it's obsolete. Using WSL_v5.
     - nlreturn # nlreturn is disabled, since it's duplicated by wsl_v5.return check.
     - ireturn  # ireturn is disabled, since it's not needed.
+#    - godox    # godox is disabled to allow TODO comments for unimplemented functionality.
```

Comment:
Commented `godox` in `disabled` section enables it. Is it the desired behavior? Just to make sure )

Reply:
fixed